### ISSUE TITLE
Validation issue

### ISSUE THREAD

**Issue by smartinez290** [haffo/hit-iz-tool#255]
Per Rob: Should we fail like this? There is no indication of the issue. Seems to me it should just be an expected segment error and keep validating.
![validationisue](https://cloud.githubusercontent.com/assets/14842422/12828363/5ba0ea5a-cb52-11e5-8e29-037cd136fc14.png)

**Comment by smartinez290** [comment]
Hi Rob,

I'm aware of this error and fixed it in the 1.0.2 version.
I already fixed it before (typo error) but somehow it showed again (I guess it was a versioning problem while pulling from Git)
It's supposed to trigger a Content Spec Error (this is a non-blocking error).

To explain what the Spec Error is :
It's related to the Dynamic Mapping and the Constraints File. Some Fields like OBX-5 have a variable Datatype. In this case the Datatype can be either complex or simple. The constraint is targeting an OBX-5 with a complex datatype and looking for the component OBX-5.x, but since the Datatype is simple we can't resolve the path and that leads to a Spec Error (because the path is specified within our resource, which is the Constraint File generated from TCAMT).

The validation should not fail however and I took care of it in this version.

Thanks,
Hossam.
